src/Microphysics2M.jl

```diff
@@ -46,22 +46,85 @@ end
 # mean terminal velocity of raindrops, and rain evaporation rates from Seifert and Beheng 2001
 
 """
-    raindrops_limited_vars(pdf_r, q_rai, ρ, N_rai)
+    pdf_cloud(pdf_c, qₗ, ρₐ, Nₗ)
+
+ - `pdf_c` - a struct with SB2006 cloud droplets size distribution parameters
+ - `qₗ` - cloud water specific humidity
+ - `ρₐ` - air density
+ - `Nₗ` cloud droplet number concentration
+
+    Returns the mean mass of cloud droplets xc [μg],
+    a multiplier needed to convert xc to base SI units χ [-],
+    and the two cloud droplet distribution parameters
+    Ac [m^-3, μg^-3] and Bc [μg^-1].
+"""
+function pdf_cloud(
+    pdf_c::CMP.CloudParticlePDF_SB2006{FT},
+    qₗ::FT,
+    ρₐ::FT,
+    Nₗ::FT,
+) where {FT}
+
+    (; νc, μc) = pdf_c
+    ϵ_q = FT(1e-12)  # TODO
+    ϵ_N = FT(10)     # TODO
+
+    α = 9 # convert mean droplet mass from kg to μg
+
+    if qₗ < ϵ_q || Nₗ < ϵ_N
+        return (xc = FT(0), Ac = FT(0), Bc = FT(0))
+    else
+        xc = qₗ * ρₐ / Nₗ * FT(10)^α                                             # μg
+        Bc = (xc * SF.gamma((νc + 1) / μc) / SF.gamma((νc + 2) / μc))^(-μc)      # 1/μg
+        Ac = μc * Nₗ * Bc^((νc + 1) / μc) / SF.gamma((νc + 1) / μc)              # 1/m3 1/μg3
+        return (; xc, α, Ac, Bc)
+    end
+end
+
+"""
+    pdf_rain(pdf_r, q_rai, ρ, N_rai)
 
  - `pdf_r` - a struct with SB2006 raindrops size distribution parameters
  - `q_rai` - rain water specific humidity
  - `ρ` - air density
- - `N_rai` raindrops number density
+ - `N_rai` rain drop number concentration
 
-    Returns the mean mass of raindrops, xr, the rate parameter of the assumed size distribution
-    of raindrops (based on drops diameter), λr, limited within prescribed ranges, and the two 
-    rain distribution parameters, A and B.
+    Returns the mean mass of rain drops xr,
+    the rate parameter of the assumed size distribution of raindrops
+    (as a function of drop diameter) λr, limited within prescribed ranges,
+    and the two rain distribution parameters, A and B.
```

Comment:
It would be helpful to write the distribution in the doc to see what A and B represent.